CMakeLists.txt

```diff
@@ -21,10 +21,12 @@ INCLUDE(cmake/boost.cmake)
 INCLUDE(cmake/lapack.cmake)
 INCLUDE(cmake/cpack.cmake)
 INCLUDE(cmake/eigen.cmake)
+INCLUDE(cmake/msvc-specific.cmake)
 
 SET(PROJECT_NAME roboptim-core)
 SET(PROJECT_DESCRIPTION "RobOptim Core")
 SET(PROJECT_URL "http://github.com/roboptim/roboptim-core")
+SET(PROJECT_DEBUG_POSTFIX "_d")
```

Comment:
Note for myself: this will change the behavior for Linux users, so this will need to be double-checked on Linux, and explained in the release changelog.